server/engine/src/main/antlr4/org/eclipse/lsp/db2/parser/Db2SqlParser.g4

```diff
@@ -47,7 +47,7 @@ result_set_locator_variable: dbs_level_01 entry_name host_variable_usage result_
 
 tableLocators_variable: dbs_host_var_levels entry_name host_variable_usage tableLocators;
 
-lob_xml_host_variables: dbs_host_var_levels entry_name  host_variable_usage xml_as (lobWithSize | xml_lobNO_size);
+lob_xml_host_variables: dbs_host_var_levels entry_name host_variable_usage xml_as (lobWithSize | xml_lobNO_size) host_variable_array_times?;
```

Comment:
I think there should be a change to the visitors to produce that correct varibles.
```
       04  VAR                              OCCURS 12345.              
         49 VAR-LENGTH                     PIC S9(9) COMP-5.           
         49 VAR-DATA                       PIC X(10).                  
      *04 VAR USAGE IS SQL TYPE IS XML AS CLOB (10) OCCURS 12345 TIMES.
```

Reply:
Addressed in latest commit.